### ISSUE TITLE
`UserWarning` when applying `Normalize`

### ISSUE THREAD

**Issue by StrausMG** [pytorch/vision#804]
Hi!
pytorch's version: 1.0.1.post2
torchvision's version: 0.2.2.post3

Sorry for the formatting, but you need to scroll the snippet to the right in order to see the warning.

```python
>>> from torch import tensor
>>> from torchvision.transforms import Normalize
>>> n = Normalize(tensor([1.0, 2.0, 3.0]),  tensor([1.0, 2.0, 3.0]))
>>> t = tensor([1.0, 2.0, 3.0]).view(3, 1, 1)
>>> n(t)
/home/ygorishniy/miniconda3/envs/x/lib/python3.6/site-packages/torchvision/transforms/functional.py:206: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  mean = torch.tensor(mean, dtype=torch.float32)
/home/ygorishniy/miniconda3/envs/x/lib/python3.6/site-packages/torchvision/transforms/functional.py:207: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  std = torch.tensor(std, dtype=torch.float32)
Out[17]: 
tensor([[[0.]],
             [[0.]],
             [[0.]]])
```

**Comment by ekagra-ranjan** [comment]
It should be in the dimension of a valid image ie: CxHxW. See the below snippet:
```python
from torch import tensor
from torchvision.transforms import Normalize
n = Normalize(tensor([1.0, 2.0, 3.0]),  tensor([1.0, 2.0, 3.0]))
t = tensor([1.0, 2.0, 3.0]).view(3, 1, 1)
n(t)
```

**Comment by StrausMG** [comment]
> Your tensor `n` should be in the dimension of a valid image ie: C x H x W. See the below snippet:
> 
> ```python
> from torch import tensor
> from torchvision.transforms import Normalize
> n = Normalize(tensor([1.0, 2.0, 3.0]),  tensor([1.0, 2.0, 3.0]))
> t = tensor([1.0, 2.0, 3.0]).view(3, 1, 1)
> n(t)
> ```

Sorry, that's exactly what I meant in the issue above. I fixed my snippet, now it is correct and still causes the warning.

**Comment by fmassa** [comment]
Hi,

Thanks for opening the issue.

The fix is to replace `torch.tensor` with `torch.as_tensor` in https://github.com/pytorch/vision/blob/6eac421ca2ea1ca52f49e10d7ac191c0d6900df1/torchvision/transforms/functional.py#L206-L207

Could you send a PR fixing this?
